images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -87,19 +96,37 @@ function Install-JavaFromAdoptOpenJDK {
     New-Item -ItemType File -Path $javaVersionPath -Name "$Architecture.complete" | Out-Null
 }
 
-$jdkVersions = (Get-ToolsetContent).java.versions
+$jdkVendors = (Get-ToolsetContent).java.vendors.name
```

Comment:
I think we should remove .name property and don't filter at line 104.

```
$jdkVendors = (Get-ToolsetContent).java.vendors
$jdkVersionsToInstall = $jdkVendor.versions
```

Reply:
Yes, it is hardcoded now for Adopt so filtering is not needed anymore